utils/goutils_test.go

```diff
@@ -120,34 +96,32 @@ func TestGetDependenciesListWithIgnoreErrors(t *testing.T) {
 
 func testGetDependenciesList(t *testing.T, testDir string) {
 	log := NewDefaultLogger(ERROR)
-	gomodPath := filepath.Join("testdata", "mods", testDir)
-	err := os.Rename(filepath.Join(gomodPath, "go.mod.txt"), filepath.Join(gomodPath, "go.mod"))
+	goModPath := filepath.Join("testdata", "mods", testDir)
+	err := os.Rename(filepath.Join(goModPath, "go.mod.txt"), filepath.Join(goModPath, "go.mod"))
 	assert.NoError(t, err)
 	defer func() {
-		err := os.Rename(filepath.Join(gomodPath, "go.mod"), filepath.Join(gomodPath, "go.mod.txt"))
+		err := os.Rename(filepath.Join(goModPath, "go.mod"), filepath.Join(goModPath, "go.mod.txt"))
 		assert.NoError(t, err)
 	}()
-	err = os.Rename(filepath.Join(gomodPath, "go.sum.txt"), filepath.Join(gomodPath, "go.sum"))
+	err = os.Rename(filepath.Join(goModPath, "go.sum.txt"), filepath.Join(goModPath, "go.sum"))
 	assert.NoError(t, err)
 	defer func() {
-		err = os.Rename(filepath.Join(gomodPath, "go.sum"), filepath.Join(gomodPath, "go.sum.txt"))
+		err = os.Rename(filepath.Join(goModPath, "go.sum"), filepath.Join(goModPath, "go.sum.txt"))
 		assert.NoError(t, err)
 	}()
-	originSumFileContent, _, err := getGoSum(gomodPath, log)
-	err = os.Rename(filepath.Join(gomodPath, "test.go.txt"), filepath.Join(gomodPath, "test.go"))
+	originSumFileContent, _, err := getGoSum(goModPath, log)
+	err = os.Rename(filepath.Join(goModPath, "test.go.txt"), filepath.Join(goModPath, "test.go"))
 	assert.NoError(t, err)
 	defer func() {
-		err := os.Rename(filepath.Join(gomodPath, "test.go"), filepath.Join(gomodPath, "test.go.txt"))
+		err := os.Rename(filepath.Join(goModPath, "test.go"), filepath.Join(goModPath, "test.go.txt"))
 		assert.NoError(t, err)
 	}()
-	actual, err := GetDependenciesList(filepath.Join(gomodPath), log)
-	if err != nil {
-		t.Error(err)
-	}
+	actual, err := GetDependenciesList(filepath.Join(goModPath), log)
+	assert.NoError(t, err)
 
 	// Since Go 1.16 'go list' command won't automatically update go.mod and go.sum.
-	// Check that we rollback changes properly.
-	newSumFileContent, _, err := getGoSum(gomodPath, log)
+	// Check that we roll back changes properly.
+	newSumFileContent, _, err := getGoSum(goModPath, log)
 	if !reflect.DeepEqual(originSumFileContent, newSumFileContent) {
```

Comment:
Let's remove the reflection here. We can use a simple assert.Equals with the text below as the message.